src/db.c

```diff
@@ -595,21 +595,23 @@ void signalFlushedDb(int dbid, int async) {
 /* Return the set of flags to use for the emptyDb() call for FLUSHALL
  * and FLUSHDB commands.
  *
- * Currently the command just attempts to parse the "ASYNC" option. It
- * also checks if the command arity is wrong.
+ * sync: flushes the cache in an sync manner.
+ * async: flushes the cache in an async manner.
```

Comment:
I would replace cache with database. I'm guessing this was taken from the script cache comment.